src/util.h

```diff
@@ -69,15 +70,50 @@ inline std::string _(const char* psz)
 void SetupEnvironment();
 bool SetupNetworking();
 
+enum LogFlags : uint32_t {
```

Comment:
idea/nit: Would be easier to maintain if it was possible to define LogFlags and CLogCategoryDesc in the same section...